diff --git a/QuEST/CMakeLists.txt b/QuEST/CMakeLists.txt
index 20c1ff9..9ecb029 100644
--- a/QuEST/CMakeLists.txt
+++ b/QuEST/CMakeLists.txt
@@ -284,7 +284,7 @@ endif()
 add_subdirectory(src)
 
 if (GPUACCELERATED)
-    cuda_add_library(QuEST SHARED
+    cuda_add_library(QuEST STATIC
         ${QuEST_SRC}
     )
 elseif (WIN32)
@@ -292,7 +292,7 @@ elseif (WIN32)
         ${QuEST_SRC}
     )
 else ()
-    add_library(QuEST SHARED
+    add_library(QuEST STATIC
         ${QuEST_SRC}
     )
 endif()
